app/Community/Enums/ArticleType.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Community\Enums;
+
+abstract class ArticleType
+{
+    public const Game = 1; // TODO commentable_type = game
+
+    public const Achievement = 2; // TODO commentable_type = achievement
+
+    public const User = 3; // TODO commentable_type = user
+
+    public const News = 4; // TODO commentable_type = news
+
+    public const Activity = 5;
+
+    public const Leaderboard = 6; // TODO commentable_type = leaderboard
+
+    public const AchievementTicket = 7; // TODO commentable_type = ticket
+
+    public const Forum = 8; // TODO ??? used for email notifications, not for comments
+
+    public const UserModeration = 9; // TODO migrate to audit log
```

Comment:
I'm not sure how to interpret the ArticleType comments - should I add additional infos to the inline comments?

Reply:
This would be similar to achievements. Players can comment on the achievements, and audit log entries can be written to the achievement comments. My comment is simply that you can't migrate _everything_ to the audit log. There are still going to be comments matching this ArticleType that aren't audit entries.